README.md

```diff
@@ -1,2 +1,30 @@
-# Interview
-A repo for interviewing pw candidates
+# PatientWisdom Pair Programming Interview
+
+Here at PatientWisdom, we support each other as a team. We're constantly faced with tasks that require us to work together to accomplish them, whether it's pair programming, reviewing code or getting opinions about decisions before they're made. This interview is designed to mimic a pair programming session with other engineers on the team.
+
+The way we organize our work is through a story based workflow. All development work is split into stories, bug fixes and chores. Before the onsite interview, we'd like to have you build the foundation for the exercises you'll work though onsite with the engineering team. We've outlined a story below to get you started.
+
+Please clone this repo and use it as a starting point for your development work. Instructions for how to get up and running are below.
+
+## Background/Epic
+
+PatientWisdom would like to add a news feed feature to the platform to assist administrators in communicating large-scale changes to their staff members. These changes could be a wide-range of possibilities including: policy updates, facility maintenance or construction, report results, etc. The system should be flexible enough to allow a hospital administrator to input a variety of different types of content to be displayed in the newsfeed. To keep things simple, there will be no user accounts or hospital-specific requirements.
+
+### Story 1
+As a healthcare team member
+I want to be able to view newsfeed items that my administrator posts
+So that I can be informed about changes at the hospital
+
+Acceptance Criteria
+---
+- [ ] There is a page that displays a table of newsfeed articles (just titles)
+- [ ] Clicking on the title of an article brings me to the article's content page
+  - [ ] The article content page consists of a post title and a blob of text
+- [ ] There is a 'Back to Newsfeed' button at the top of every article that takes me back to the main list page
+
+Development Notes
+---
```

Comment:
arguably, this leaves open the door for them to just make a pure rails app which serves routes. Do we want to give them the reign to do that, or do we really want to see them use React, so we should setup that expectation?

Reply:
done 65d0751a3d71d64c0df0627fd1f7b8e6b8acf726